src/components/molecules/ReactionsBar/ReactionsBar.scss

```diff
@@ -0,0 +1,37 @@
+@import "scss/constants";
+
+.ReactionsBar {
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  margin: 5px 0;
+  width: 100%;
+
+  &--mute-button {
+    display: flex;
+    align-items: center;
+    font-size: 20px;
+    cursor: pointer;
+  }
+
+  &--leave-seat-button {
+    display: inline-block;
+    font-weight: 700;
```

Comment:
Check `constants` for a value.

Reply:
fixed in 9cbbfe8f7c30d7ffb04d2dd93820d4d5add71ec4